app/models/product.rb

```diff
@@ -8,7 +8,7 @@ class Product < ApplicationRecord
   validates :quantity, presence: true
   validates :price, presence: true
   validates :price_at, presence: true
-  validates :assets, length: { minimum: 1, maximum: 4}
+  validates :assets, length: { minimum: 1, maximum: 4 }
 
   def sale_product
     self.update_attribute(:status, 'selling')
```

Comment:
`selling` should be defined const in model product
`SELLING = 'selling'.freeze`
`self.update_attribute(:status, SELLING)`